src/server.h

```diff
@@ -3678,6 +3678,9 @@ unsigned long LFUDecrAndReturn(robj *o);
 int performEvictions(void);
 void startEvictionTimeProc(void);
 
+/* delete unowned keys from database at server start up after loading persistence files. */
```

Comment:
In Valkey project, we generally put the code comment in `.c` file rather than the `.h` file.